### ISSUE TITLE
winappdriver can not find element by xpath on windows(1809)

### ISSUE THREAD

**Issue by Emilymcwu** [microsoft/WinAppDriver#1020]
The same code，winappdriver can find element by xpath on windows(1903)，can not find element by xpath on windows（1809）

My xpath code：
String liveIdXpath = "/Pane[@ClassName=\"#32769\"][@Name=\"桌面 1\"]/Window[@Name=\"教师端\"][@AutomationId=\"LiveListDialog\"]/Group/Custom/Group/Group[position()=2]/Group[position()=2]/Group/Group/Group/Group[position()=1]/Text";

**Comment by uday2503** [comment]
@Emilymcwu is the issue resolved for you.
If yes could you please tell me how?

**Comment by khanhnd25** [comment]
Yes. I was able to make it work..I did this I did some trial and error and I removed the 1st token of the above XPATH.

Changed /Pane[@Name='Desktop 1']/Window[@Name='FusionReg-MockUI']/Window[@Name='FusionReg-MockUI']/RadioButton[@Name='QA']

to

/Window[@Name='FusionReg-MockUI']/Window[@Name='FusionReg-MockUI']/RadioButton[@Name='QA'] and it worked.
